src/toil/jobStores/abstractJobStore.py

```diff
@@ -100,6 +100,10 @@ def __init__(self, locator):
             "the job store with 'toil clean' to start the workflow from scratch" % locator)
 
 
+class JobStoreBucketExistsException(Exception):
+    def __init__(self, message):
+        super(JobStoreBucketExistsException, self).__init__(message)
```

Comment:
I was thinking of it but there are two possible messages that make it through hence the format i used.